src/java/map/TerrainMaterials.java

```diff
@@ -0,0 +1,30 @@
+package map;
+
+public strictfp class TerrainMaterials {
+
+    // engine limitations - must stay 9 and 10 always
+    public final int TERRAIN_TEXTURE_COUNT = 10;
+    public final int TERRAIN_NORMAL_COUNT = 9;
+
+    public String[] texturePaths = new String[TERRAIN_TEXTURE_COUNT];
+    public float[] textureScales = new float[TERRAIN_TEXTURE_COUNT];
+    public String[] normalPaths = new String[TERRAIN_NORMAL_COUNT];
+    public float[] normalScales = new float[TERRAIN_NORMAL_COUNT];

```

Comment:
encapsulation please, private and @Data or @getter / @setter or even only for specific fields